kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/k8s/KubeClient.java

```diff
@@ -187,10 +195,28 @@ public Pod getPod(String name) {
      * Create or replace deployment deployment.
      *
      * @param deployment the deployment
-     * @return the deployment
      */
-    public Deployment createOrReplaceDeployment(Deployment deployment) {
-        return client.apps().deployments().inNamespace(deployment.getMetadata().getNamespace()).resource(deployment).create();
+    public void createOrReplaceDeployment(Deployment deployment) {
+        client.apps().deployments().inNamespace(deployment.getMetadata().getNamespace()).resource(deployment).create();
```

Comment:
the name of the method disagrees with the implementation.  You could use:

```        client.apps().deployments().inNamespace(deployment.getMetadata().getNamespace()).resource(deployment).createOr(NonDeletingOperation::replace);
```

Reply:
done